resources/getApiKey_test.go

```diff
@@ -0,0 +1,46 @@
+package resources
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetApiKey(t *testing.T) {
```

Comment:
I, personally, am more of a fan of having separate tests for the individual cases. Could you, please, re-work this one to match the "given-when-then" approach that you use in the integration tests?